Demo/src/main/java/com/paypal/android/ui/features/FeaturesView.kt

```diff
@@ -53,8 +58,8 @@ fun FeaturesView(
     LazyColumn(
         modifier = Modifier
             .background(MaterialTheme.colorScheme.surface)
-            .padding(horizontal = UIConstants.paddingMedium)
-            .fillMaxSize()
+            .fillMaxSize(),
+        contentPadding = PaddingValues(bottom = UIConstants.paddingMedium)
```

Comment:
Some clipping occurs on the bottom of the `FeaturesView` scroll view content area. The `contentPadding` update here ensures that all buttons are equally visible.